imager/imager_profile/models.py

```diff
@@ -1,3 +1,36 @@
+from django.utils.encoding import python_2_unicode_compatible
 from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+PHOTOGRAPHY_TYPES = [
+    ('portrait', 'Portrait'),
+    ('landscape', 'Landscape'),
+]
+
+
+class ActiveManager(models.Manager):
+    def get_queryset(self):
+        qs = super(ActiveManager, self).get_queryset()
+        return qs.filter(user__is_active__exact=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL,
+                                on_delete=models.CASCADE,
+                                related_name='profile',
+                                null=False)
+    location = models.CharField(default='', max_length=255)
+    bio = models.TextField(default='')
+    camera = models.TextField(default='')
+    photography_type = models.CharField(max_length=30, default='portrait',
+                                        choices=PHOTOGRAPHY_TYPES)
+    friends = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name='friend_of')
+    objects = models.Manager()
```

Comment:
Why do you override the default objects manager that django models provide for you already? And why do you do it with the default manager?